src/Layout/redist/targets/GeneratePKG.targets

```diff
@@ -140,26 +140,42 @@
           Inputs="@(GenerateSdkPkgInputs)"
           Outputs="$(SdkPKGInstallerFile)"
           DependsOnTargets="GenerateInstallerLayout;SetupPkgInputsOutputs">
-    <ItemGroup>
-      <TemplateFiles Include="$(RedistLayoutPath)templates/**/*" />
-    </ItemGroup>
+    <!-- Copy the sdk layout into a temporary folder to be able to make changes to it. -->
+    <PropertyGroup>
+      <SdkInternalLayoutPath>$(IntermediateOutputPath)installer/</SdkInternalLayoutPath>
+    </PropertyGroup>
 
+    <!-- Create "SDK Internal" layout. -->
     <ItemGroup>
-      <ManifestFiles Include="$(RedistLayoutPath)sdk-manifests/**/*" />
+      <SdkOutputFile Include="$(OutputPath)**/*" />
     </ItemGroup>
-    
+
+    <Copy SourceFiles="@(SdkOutputFile)"
+          DestinationFiles="@(SdkOutputFile -> '$(SdkInternalLayoutPath)sdk/$(Version)/%(RecursiveDir)%(Filename)%(Extension)')"
+          UseHardLinksIfPossible="true"
```

Comment:
I'm not sure creating a separate copy is compatible with the ReplaceBundledRuntimePackFilesWithSymbolicLinks logic, can you double check?

Reply:
Yes, so this Copy was already there before, but for all verticals. I moved it into GeneratPKG.targets so that it now only applies to macOS verticals. The copy is only necessary there because of the tracking issue that is mentioned in the same file.

Also, I just noticed that the `ReplaceBundledRuntimePackFilesWithSymbolicLinks` target only runs for source-build, so unrelated to GeneratePKG today.